File: app/src/main/java/ar/edu/utn/frba/dadm/clases2018c1/clases_2018c1/persistence/db/entities/Movie.java
package ar.edu.utn.frba.dadm.clases2018c1.clases_2018c1.persistence.db.entities;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

@Entity
public class Movie {
    @PrimaryKey
    @NonNull
    public String title;

    public String year;

    public String imageUri;
}

File: app/src/main/java/ar/edu/utn/frba/dadm/clases2018c1/clases_2018c1/api/responses/Movie.kt
package ar.edu.utn.frba.dadm.clases2018c1.clases_2018c1.api.responses

import java.io.Serializable

class Movie: Serializable {

    var response: Boolean = false
    var title: String? = null
    var year: String? = null
    var poster: String? = null
    var metascore: String? = null
}